packages/fxa-auth-server/lib/routes/index.js

```diff
@@ -86,7 +89,8 @@ module.exports = function (
     totp,
     unblockCodes,
     util,
-    recoveryKey
+    recoveryKey,
+    subscriptions
```

Comment:
As mentioned above, can we add a config feature flag that says whether this feature is enabled and only add these routes if enabled? Note, this might be a good use of @philbooth's feature flagging, we could flip the feature on dynamically.

Reply:
Probably better to gate with a configuration flag on deployment, since setting up products & clients needs config tweaks too. I don't think we'd get much from dynamic flagging.

Reply:
https://github.com/mozilla/fxa/pull/819/commits/0f09f92c619439ffe6d522cc4c8a5c972b6bf70b

Reply:
Also, ref: #475, feature-flagging is not set up in the auth server yet.